src/WebJobs.Script/Config/RpcWorkerConcurrencyOptions.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public class RpcWorkerConcurrencyOptions : Hosting.IOptionsFormatter
```

Comment:
This shouldn't be Rpc specific right, since we want these worker options to apply to Http workers as well?

Reply:
Fixed.